internal/pkg/archer/project.go

```diff
@@ -18,10 +18,11 @@ func (p *Project) RequiresDNSDelegation() bool {
 
 // ProjectRegionalResources represent project resources that are regional.
 type ProjectRegionalResources struct {
-	Region         string            // The region these resources are in.
-	KMSKeyARN      string            // A KMS Key ARN for encrypting Pipeline artifacts.
-	S3Bucket       string            // S3 bucket for Pipeline artifacts.
-	RepositoryURLs map[string]string // The image repository URLs by app name.
+	Region                     string            // The region these resources are in.
+	KMSKeyARN                  string            // A KMS Key ARN for encrypting Pipeline artifacts.
+	S3Bucket                   string            // S3 bucket for Pipeline artifacts.
+	S3BucketRegionalDomainName string            // Regional domain name of S3 bucket for Pipeline artifacts.
```

Comment:
Do we really need this? It looks like the only reason we're using it is so that we can get the URL of the object we upload to S3, and pass it to CloudFormation.

I think what we can do instead, is not store this - but read the URL out of the UploadObject response. [ref](https://github.com/aws/aws-sdk-go/blob/v1.29.11/service/s3/s3manager/upload.go#L100-L112).